jest.config.js

```diff
@@ -2,6 +2,6 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-  rootDir: 'test',
+  rootDir: 'test/server',
```

Comment:
Jest tries to find test from `*.ts` files under the `rootDir` and it requires all those files contain tests. And since new `ts` util files got added under `test`, we need to point jest to only find test under `test/server`

Reply:
Ahhh, ok - good to know. Thank you 🙂